book/website/community-handbook/local-build.md

```diff
@@ -1,154 +1,184 @@
+It is useful to preview changes you have been working on as you go on your local machine.
+You can be confident that changes you have made are accurate and as intended and it will likely be quicker than waiting for a preview to be build from a pull request.
+You can replicate the build process using [Make](rr-make) and [Jupyter Book](https://jupyterbook.org/en/stable/intro.html).
 
-This is useful because it allows you to preview any changes you have made on your local machine *before* you push your changes to a remote branch.
-You can then decide if you are happy with the result and push your changes to the remote branch thus helping to keep Pull Request conversations and commit histories a bit cleaner.
-
-## Step-by-step guide
+## Prerequisites
 
 We will be using the command line throughout this guide.
-You will need to locate your "terminal" or "prompt" application on your machine.
+You will need to use a terminal emulator to follow.
 
-1. Install miniconda by following the instructions for your Operating System (Windows, MacOS, Linux) at this link: https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation
+You will also need to install Python3.
+You can check which specific version of Python3 the build uses in [netlify.toml](https://github.com/the-turing-way/the-turing-way/blob/main/netlify.toml).
+However, versions close to that version are also likely to work.
 
-2. Open your terminal app and run the `conda init` command in your terminal. You should see `(base)` in your prompt indicating that conda was successfully installed and you are now in its base environment.
-    - Note that if you are using Windows, you will need to open a program called `Anaconda prompt` instead of using `cmd`.
+Other command line tools you will need are,
 
-3. Create a new environment and install a modern version of Python into it:
+- `git`
+- `make`
 
-   ```console
-   conda create --name the-turing-way python=3.13
-   ```
+## Step-by-step Guide
 
-4. Activate the environment with:
+### Clone The Repository
 
-   ```console
-   conda activate the-turing-way
-   ```
+```console
+git clone https://github.com/the-turing-way/the-turing-way
+```
 
-   Any commands we run with Python or pip from now on will use the versions of Python and pip installed into _this_ conda env, not any others.
+````{note}
+The repository is quite large and cloning may take a long time on slower internet connections.
+You can use [partial clones](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/#).
+Specifically, focusing on blobless clones, involves utilizing the `--filter=blob:none` option in the git clone command.
 
-5. Clone _The Turing Way_ repository from GitHub to your machine using the command: 
+By using `--filter=blob:none`, the initial git clone operation downloads all reachable commits and trees, while blobs (file contents) for commits are only downloaded when performing a git checkout.
```

Comment:
I think this is an unnessary side-track, and overall I would rather like to see explaining how to work with forks.

Reply:
I think it is valuable information, especially given the size of the repo. Maybe it could be placed in a better location though.

It is someone else work, so I wouldn't have wanted to remove it.

I think changing this to include forks depends on the outcome of https://github.com/the-turing-way/the-turing-way/issues/3871.

Reply:
> It is someone else work, so I wouldn't have wanted to remove it.

Yes, fair enough.